src/WebJobs.Extensions.DurableTask/Listener/TaskOrchestrationShim.cs

```diff
@@ -134,6 +134,25 @@ public override async Task<string> Execute(OrchestrationContext innerContext, st
             return serializedOutput;
         }
 
+        private void TraceOutOfProcReplay(object result)

```

Comment:
Is this method being called anywhere? A quick search using Ctrl+F in my browser didn't find anything. I also noticed that  we have another call to TraceHelper.ProcessingOutOfProcPayload inside the GetInvocationResult method, and that version seems to have the optimizations I was asking for.

Reply:
You're right, this wasn't being used anymore. I thought Visual Studio would have complained about this, interesting 🤔